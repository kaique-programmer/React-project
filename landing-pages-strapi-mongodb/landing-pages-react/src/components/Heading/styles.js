import styled, { css } from 'styled-components';

export const Title = styled.h1`
  ${({ theme }) => css`
    color: ${light ? theme.colors.white : theme.colors.primaryBackground }
  `}
`;
