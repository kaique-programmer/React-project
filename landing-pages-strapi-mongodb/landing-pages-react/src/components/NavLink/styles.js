import styled, { css } from 'styled-components';

export const Container = styled.div`
  ${({ theme }) => css`
    display: flex;
    background: ${theme.colors.white};
    flex-flow: row wrap;

    @media ${theme.media.lteMedium} {
      flex-flow: column wrap;
      align-content: center;
    }
  `}
`;
