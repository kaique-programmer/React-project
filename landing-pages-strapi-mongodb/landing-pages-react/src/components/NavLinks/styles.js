import styled, { css } from 'styled-components';

export const Container = styled.div`
  ${({ theme }) => css`
    display: flex;
    background: #fff;
    flex-flow: row wrap;

    @media ${theme.media.lteMedium} {
      flex-flow: column wrap;
      align-content: center;
    }
  `}
`;
