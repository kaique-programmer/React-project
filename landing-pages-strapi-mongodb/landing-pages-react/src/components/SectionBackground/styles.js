import styled, { css } from 'styled-components';

const containerBackgroundActivate = (theme) => css`
  background: ${theme.colors.white};
  color: ${theme.colors.primaryBackground};
`;

export const Container = styled.div`
  ${({ theme, background }) => css`
    background: ${theme.colors.white};
    color: ${theme.colors.primaryBackground};
    ${background && containerBackgroundActivate(theme)};
    min-height: 100vh;
    display: flex;
    align-items: center;
  `}
`;
