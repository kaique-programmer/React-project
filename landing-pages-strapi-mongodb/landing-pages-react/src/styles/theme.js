const theme = {
  colors: {
    primaryBackground: 'red',
    secondaryBackground: 'blue',
  },
  fonts: {},
  spacings: {},
};

export default theme;
