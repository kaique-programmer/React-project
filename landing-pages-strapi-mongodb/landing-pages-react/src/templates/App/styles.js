import styled, { css } from 'styled-components';

export const Container = styled.div`
  ${({ props }) => css`
    background: ${props.background};
  `}
`;
