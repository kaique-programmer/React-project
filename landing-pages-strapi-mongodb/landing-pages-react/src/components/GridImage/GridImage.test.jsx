import React from 'react';
import { screen } from '@testing-library/react';
import { renderTheme } from '../../styles/render-theme';
import { GridImage } from '.';

describe('<GridImage />', () => {
  it('should render grid image', () => {
    renderTheme(<GridImage>Children</GridImage>);
    expect(screen.getByRole('heading')).toBeInTheDocument();
  });

  it('should render grid image', () => {
    const { container } = renderTheme(<GridImage/>);
    expect(container).toMatchSnapshot();
  });
});
