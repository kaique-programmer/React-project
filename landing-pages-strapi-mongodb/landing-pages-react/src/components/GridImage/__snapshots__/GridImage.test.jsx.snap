// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GridImage /> should render grid image 1`] = `
<div>
  <div
    class=""
  >
    <h1 />
  </div>
</div>
`;
